ts/Core/Axis/Axis.ts

```diff
@@ -4171,6 +4179,7 @@ class Axis {
          */
         axis.reversed = pick(options.reversed, axis.reversed);
         axis.visible = options.visible;
+        axis.panningEnabled = options.panningEnabled;
```

Comment:
As far as I can see, we don't need to make this an instance property, because it is always the same as the option. So we can just refer to the option directly instead.